go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/hashicorp/go-version v1.3.0 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mattn/go-isatty v0.0.14
+	github.com/mcuadros/go-version v0.0.0-20190830083331-035f6764e8d2
```

Comment:
This is unmaintained; please use https://github.com/coreos/go-semver instead if possible.

Reply:
Sure, will update it.